tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -296,27 +297,37 @@ public function userUpdatesLastPublicLinkShareUsingTheGraphApiWith(string $user,
 	 * @throws JsonException
 	 * @throws \GuzzleHttp\Exception\GuzzleException
 	 */
-	public function userRemovesSharePermissionOfUserFromResourceOfSpaceUsingGraphAPI(string $user, string $shareeUser, string $resourceType, string $resource, string $space): void {
-		$spaceId = ($this->spacesContext->getSpaceByName($user, $space))["id"];
+	public function userRemovesSharePermissionOfUserFromResourceOfSpaceUsingGraphAPI(
+		string $sharer,
+		string $shareType,
+		string $sharee,
+		string $resourceType,
+		string $resource,
+		string $space
+	): void {
+		$spaceId = ($this->spacesContext->getSpaceByName($sharer, $space))["id"];
 		$itemId = ($resourceType === 'folder')
-			? $this->spacesContext->getResourceId($user, $space, $resource)
-			: $this->spacesContext->getFileId($user, $space, $resource);
-		$userIdOfShareeUser = $this->featureContext->getUserIdByUserName($shareeUser);
+			? $this->spacesContext->getResourceId($sharer, $space, $resource)
+			: $this->spacesContext->getFileId($sharer, $space, $resource);
+		$userIdOfSharee = ($shareType === 'user')
+		? $this->featureContext->getUserIdByUserName($sharee)
+			: $this->featureContext->getGroupIdByGroupName($sharee);
 		$permId = GraphHelper::getSharePermissionId(
 			$this->featureContext->getBaseUrl(),
 			$this->featureContext->getStepLineRef(),
-			$user,
-			$userIdOfShareeUser,
-			$this->featureContext->getPasswordForUser($user),
+			$sharer,
+			$userIdOfSharee,
+			$this->featureContext->getPasswordForUser($sharer),
 			$spaceId,
-			$itemId
+			$itemId,
+			$shareType
 		);
 		$this->featureContext->setResponse(
 			GraphHelper::deleteSharePermission(
```

Comment:
```suggestion
			GraphHelper::removeSharePermission(
```
May be change also the function name.